src/pages/auth/login/index.tsx

```diff
@@ -26,7 +28,8 @@ export default function LoginPage({ providers, csrfToken }) {
       <div className={styles.mainSec}>
         <h2 className={styles.title}>Login</h2>
         {error && <SignInError error={error} />}
-        <form method="post" action="/api/auth/callback/credentials"> {/*TODO: save session into db*/}
+        <form method="post" action={'/api/auth/callback/credentials'}>
```

Comment:
i think we should add a login service here

Reply:
What do you mean?